packages/@o3r/eslint-config/src/public_api.ts

```diff
@@ -0,0 +1,2 @@
+import typescript from './rules/typescript';
+export default typescript;
```

Comment:
A simple `export * from './rules/typescript';` would not do the job?

Reply:
I can retry but it was not exposing a default if I remember well